### PR TITLE
Style, accessibility, content updates for headers/footers

### LINKED ISSUE #98: Header/footer updates
- Make footer look better on mobile devices
- Update header to match EDD's more closely
- The header appears to be responsive on the EDD static site (e.g., try narrowing the browser to see it change size).
- The header text (jobs, claims, etc) is justified or something, vs centered and fixed like the EDD site.

### PR DESCRIPTION
**Pull request by sharonwarner**
Resolves #98 

- Moves header/footer text to translation files
- Updates font sizes to match EDD's website
- Updates header spacing to match EDD's website
- Makes footer mobile accessible

headerFooterUpdates

### CODE REVIEW

File: src/client/components/Header/__snapshots__/index.test.js.snap [hunk 1]
```diff
@@ -39,13 +38,16 @@ exports[`<Header /> renders the header component 1`] = `
...
           src="images/home.svg"
           width="15"
         />
          
-        <span>
-          Home
+        <span
+          className="text"
+        >
+          header.home
```
Comment (kalvinwang) [inline src/client/components/Header/__snapshots__/index.test.js.snap:50]: hm seems like this didn't render the translation in the snapshot, isn't it supposed to?

File: src/client/components/Header/__snapshots__/index.test.js.snap [hunk 2]
```diff
@@ -59,21 +61,25 @@ exports[`<Header /> renders the header component 1`] = `
...
+            className="icon"
+            height="15"
+            src="images/key.svg"
+            width="15"
+          />
+           
+          <span
+            className="text"
+          >
+            header.login
```
Comment (kalvinwang) [inline src/client/components/Header/__snapshots__/index.test.js.snap:76]: same q

File: src/client/components/Header/__snapshots__/index.test.js.snap [hunk 3]
```diff
@@ -107,10 +113,8 @@ exports[`<Header /> renders the header component 1`] = `
         disabled={false}
         href="https://edd.ca.gov/jobs.htm"
       >
-        <span
-          className="text-dark"
-        >
-          Jobs
+        <span>
+          header.jobs
```
Comment (kalvinwang) [inline src/client/components/Header/__snapshots__/index.test.js.snap:117]: etc

File: src/client/components/Header/__snapshots__/index.test.js.snap (hunk 1 above)
Comment (sharonwarner) [reply src/client/components/Header/__snapshots__/index.test.js.snap:50]: interesting! it looks like all of our snapshot tests are broken in this way. i'm going to merge this PR and fix that issue separately.

**Comment by idealisms** [comment]
I'm noticing on mobile, that the footer is overlapping the content. The 120px isn't necessarily enough space based on wrapping of the links in the footer.

Not sure what the best way to handle that is. I guess we can use a bigger margin-bottom on body? Or refactor to not use position absolute for the footer.

**Pull request merged**